### ISSUE TITLE
Parse .NET headers

### ISSUE THREAD

**Issue by jbremer** [lief-project/LIEF#64]
Hi,

Just stumbled upon this - looks good! Just wanted to let you know that if you add .NET header parsing to LIEF that I'll replace our current `pefile` & `pype32` dependencies with LIEF in Cuckoo Sandbox (assuming that deployment on different architectures is successful and all unit tests pass etc). Let me know! :-)

Jurriaan

**Comment by romainthomas** [comment]
Yes it's planned but don't know yet if it will be available for the ``v0.8.0`` or after

**Comment by jbremer** [comment]
Sweet :) take your time.
